signals/commands/things/src/main/java/org/eclipse/ditto/signals/commands/things/ThingErrorResponse.java

```diff
@@ -168,7 +170,23 @@ public static ThingErrorResponse fromJson(final ThingErrorRegistry thingErrorReg
                         .build());
         final JsonObject payload = jsonObject.getValueOrThrow(ThingCommandResponse.JsonFields.PAYLOAD).asObject();
 
-        final DittoRuntimeException exception = thingErrorRegistry.parse(payload, dittoHeaders);
+        DittoRuntimeException exception;
+        try {
+            exception = thingErrorRegistry.parse(payload, dittoHeaders);
+        } catch (final Exception e) {
+            final int status = jsonObject.getValue(CommandResponse.JsonFields.STATUS).orElse(500);
+            final String errorCode =
+                    payload.getValue(DittoRuntimeException.JsonFields.ERROR_CODE).orElse("unknown:unknown");
+            final String errorMessage =
+                    payload.getValue(DittoRuntimeException.JsonFields.MESSAGE).orElse("An unknown error occurred");
+            final String errorDescription = payload.getValue(DittoRuntimeException.JsonFields.DESCRIPTION).orElse("");
+            exception =
+                    DittoRuntimeException.newBuilder(errorCode,
+                            HttpStatusCode.forInt(status).orElse(HttpStatusCode.INTERNAL_SERVER_ERROR))
+                            .message(errorMessage)
+                            .description(errorDescription)
```

Comment:
Any reason why the `dittoHeaders` are not passed into the builder here?
`thingErrorRegistry.parse(payload, dittoHeaders)` also passes the DittoHeaders ..
Should be set I think.